polkadot-protocol/onchain-governance/origins-tracks.md

```diff
@@ -0,0 +1,119 @@
+
+## Track Parameters
+
+While the Track structure is consistent across all `Origins`, the parameters for each `Track` can vary. These parameters include:
+
+1. **Maximum deciding capacity** - defines the limit on the number of referenda that can be decided simultaneously, corresponding to the number of tracks within each origin
+2. **Decision deposit** - specifies the funds required to be deposited for a proposal to enter the decision period (additional requirements must also be met to proceed)
+3. **Preparation period** - represents the minimum voting duration required before a proposal can enter the decision period, provided capacity and deposit conditions are satisfied
+4. **Decision period** - the time frame during which the outcome of a proposal is determined
+5. **Confirmation period** - denotes the minimum duration during which approval and support thresholds must be sustained for a proposal to proceed to the enactment period. The confirmation period should begin before the decision period ends
+6. **Voting period** - encompasses the entire duration during which voting is permitted on a referendum, including the preparation, decision, and confirmation periods
+7. **Minimum enactment period** - establishes the minimum wait time before the approved changes are implemented
+8. **Approval curve** - illustrates the minimum percentage of aye votes required over time within the decision period. The approval percentage is calculated as the proportion of aye votes (adjusted for conviction) relative to the total votes (`aye`, `nay`, and `abstained`)
+9. **Support curve** - depicts the minimum percentage of total votes in favor of a proposal over time during the decision period. Support percentage is defined as the proportion of aye and abstained votes (without conviction) relative to the system's total possible votes (i.e., the active issuance)
+
+For instance, a runtime upgrade (requiring a [`set_code`](https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/dispatchables/fn.set_code.html){target=\_blank} call) has far-reaching implications for the ecosystem compared to the approval of a treasury tip (via a [`report_awesome`](https://paritytech.github.io/polkadot-sdk/master/pallet_tips/pallet/dispatchables/fn.report_awesome.html){target=\_blank} call). As a result, distinct `Origins` are assigned for these actions, each with predefined parameters such as deposits, support, approval thresholds, and minimum enactment periods tailored to their specific requirements.
+
+!!!note
+    More information about terminology and parameters can be found on the [OpenGov Origins Wiki](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins){target=\_blanck} page.
+
+In the table below you can find all `Origins` and its `Tracks` on the Polkadot chain:
+
+| **ID** | **Origin**               | Max Deciding | Decision Deposit | Prepare Period | Decision Period | Confirm Period | Min Enactment Period | Min Approval      | Min Support       |
```

Comment:
@eshaben @dawnkelly09 How do you both feel about this table? I think we made an effort to replace tables in the past or something like that, but not sure

IMO, I don't see all the information in this table as necessary for devs, so maybe we can restructure this a bit and just leave fields like `Decision Deposit`, `Min Enactment Period`, and `Min Approval`; for the rest, we can add a link to these sections of the Polkadot wiki: https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins#root

WDYT?

Reply:
Yeah, I think @dawnkelly09 mentioned something about cleaning that up based on Radha's feedback on a similar PR. So she's going to take a look at this and remove some of that info when she gets some time

Reply:
@nhussein11 I'll remove `### Examples of Origins in OpenGov`, but content-wise, do you think page contains enough information about Origins and Tracks? (I am talking purely about the Learn section) 

@dawnkelly09 please don't butcher it too much :')